thirdeye/thirdeye-frontend/app/pods/manage/alert/explore/route.js

```diff
@@ -379,27 +337,12 @@ export default Route.extend({
 
     // Cancel all pending concurrency tasks in controller
     if (isExiting) {
+      this.get('loadAnomalyData').cancelAll();
```

Comment:
I tested this locally and it seems to help with quicker transitions while large datasets are still loading.